tests/nnapi/nnapi_gtest.skip.armv7l-linux.acl_cl

```diff
@@ -47,6 +47,7 @@ GeneratedTests.gather_float16_8
 GeneratedTests.log_4D_float_nnfw
 GeneratedTests.log_dynamic_nnfw
 GeneratedTests.logical_not
+GeneratedTests.logical_not_dynamic_nnfw
```

Comment:
in tests/nnapi/nnapi_gtest.skip.aarch64-linux.acl_neon,
```
GeneratedTests.logical_not_1D_nnfw
GeneratedTests.logical_not_4D_nnfw	
GeneratedTests.logical_not_4D_nnfw
GeneratedTests.logical_not_dynamic_nnfw
```
but here, order seems changed. I don't know which should be first between num or alpha, but IMHO we need some consistency. :-)